src/Comments.jsx

```diff
@@ -34,6 +38,23 @@ export const Comments = ({ article_id }) => {
     setcurrentPage(currentPage + 1);
   };
 
+  const removeComment = (commentToDelete) => {
```

Comment:
Could we have some UI to show the delete was successful and also if it is mid-delete?